plugin/hover.py

```diff
@@ -409,3 +409,10 @@ def _update_views_async(self, enable: bool) -> None:
                         session_view.view.settings().set(SHOW_DEFINITIONS_KEY, False)
                     else:
                         session_view.reset_show_definitions()
+
+
+class LspCopyTextCommand(sublime_plugin.TextCommand):
+    def run(self, edit, text: str) -> None:
+        sublime.set_clipboard(text)
+        text_length = len(text)
+        sublime.status_message(f"Copied {text_length} character{'s' if text_length > 1 else ''}")
```

Comment:
I think the diagnostic message will always be longer than 1 character, so not sure if this is needed.